src/main/java/com/hcl/appscan/jenkins/plugin/scanners/Scanner.java

```diff
@@ -38,7 +40,9 @@ public String getTarget() {
 	}
 	
 	public abstract Map<String, String> getProperties(VariableResolver<String> resolver) throws AbortException;
-	
+
+	public abstract void validations(IAuthenticationProvider authProvider, Map<String, String> properties, IProgress progress) throws AbortException;
```

Comment:
In DynamicAnalyzer class we are doing the URL validity check & for that we need to call the getProxy() method from authProvider class. Instead of passing the IAuthenticationProvider, I will pass the JenkinsAuthenaticationProvider itself.